OreNoTask/spec/models/task_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Taskモデルのテスト', type: :model do
+  describe '正常系' do
+    it '全項目入力' do
+      task = Task.new(
+        name: '最初のタスク', description: '説明文', start_at: '2021/09/01 10:00', due_date_at: '2021/09/02 11:00',
+      )
+      expect(task).to be_valid
+    end
+
+    it 'descriptionが空欄' do
+      task = Task.new(
+        name: '最初のタスク', description: '', start_at: '2021/09/01 10:00', due_date_at: '2021/09/02 11:00',
+      )
+      expect(task).to be_valid
+    end
+  end
+
+  describe 'バリデーションのテスト' do
+    context 'nameカラム' do
+      it '空欄でないこと' do
+        task = Task.new(
+          name: '', description: '説明文', start_at: '2021/09/01 10:00', due_date_at: '2021/09/02 11:00',
+        )
+        expect(task.valid?).to eq false
+      end
```

Comment:
describeやcontext、itなどの使い方が少し気になりました
describeはテスト対象など”何”を
contextは条件を
itは求める結果を書いてほしいです
また、letなどを用いると以下のような繰り返し書かれているものをリファクタできるかと思うのでぜひ使ってみてください
```ruby
        task = Task.new(
          name: '', description: '説明文', start_at: '2021/09/01 10:00', due_date_at: '2021/09/02 11:00',
        )
```

```suggestion
  describe 'バリデーションのテスト' do
    let(:task) { Task.new(name: name, description: description, start_at: start_at, due_date_at: due_date_at) }
    let(:name) { '最初のタスク' }
    let(:description) { '説明文' }
    let(:start_at) { '2021/09/01 10:00' }
    let(:due_date_at) { '2021/09/02 11:00' }
    describe 'nameカラムは' do
      let(:name) { '' } # ここでnameを上書いています。あまり上書くのは良くないとも言われたりもしてますが…
      context '空欄のとき' do
        it 'エラーになること' do
          expect(task.valid?).to eq false
        end
      end
```

Reply:
構成を変更し、letを使うようにしました！